courses/api.py

```diff
@@ -66,3 +74,39 @@ def get_user_enrollments(user):
         non_program_runs=non_program_run_enrollments,
         past_non_program_runs=past_non_program_run_enrollments,
     )
+
+
+def generate_program_certificate(user, program):
```

Comment:
Shouldn't this be in `courses/utils.py`? I don't see anything API specific in this method.